network/src/validation.rs

```diff
@@ -804,6 +843,15 @@ impl<P: ProvideRuntimeApi + Send, E, N, T> SessionDataFetcher<P, E, N, T> where
 		});
 		PoVReceiver { outer: rx, inner: None }
 	}
+
+	pub fn fetch_erasure_chunk(&self, relay_parent: Hash, candidate_hash: Hash, chunk_id: u64) -> ChunkReceiver {
```

Comment:
suggest changing the argument types here to type aliases instead, as mentioned here https://github.com/paritytech/polkadot/pull/345/files#r308998575